src/containers/App/App.js

```diff
@@ -45,7 +45,18 @@ class App extends Component {
       params: { rippledUrl = null },
     } = match;
     const rippledHost = rippledUrl ?? process.env.REACT_APP_RIPPLED_HOST;
-    this.socket = new XrplClient([`wss://${rippledHost}:${process.env.REACT_APP_RIPPLED_WS_PORT}`]);
+    const wsUrls = [];
+    if (rippledHost.includes('wss')) {
+      wsUrls.push(rippledHost);
+    } else if (rippledHost.includes(':')) {
+      wsUrls.push(`wss://${rippledHost}`);
+    } else {
+      wsUrls.push.apply(wsUrls, [
+        `wss://${rippledHost}:${process.env.REACT_APP_RIPPLED_WS_PORT}`,
+        `wss://${rippledHost}`,
```

Comment:
Is this just a for sidechains?

Reply:
The changes are mainly for sidechains, yes - we can ensure for the other versions that the port isn't included in the URL. But this allows us to e.g. support `xrplcluster.com` as a backup URL, since that doesn't take a port.